### ISSUE TITLE
Fuzzy Commands might be nice

### ISSUE THREAD

**Issue by RyanGreenup** [RyanGreenup/cadmus#10]
So for example:

```bash
echo "
cadmus find
cadmus search
cadmus tags
cadmus tags create
cadmus tags filter
cadmus tools mount
cadmus export
cadmus convert
cadmus misc
cadmus publish
cadmus preview
cadmus help" > /tmp/mycommands
```
and then

```bash
eval $(cat /tmp/mycommands | sk --height 40%)
```

This could be tied to something like `cadmus misc fuzzy` and users could alias it to something like `caz`, see for example this Gif:

![fuzzy](https://user-images.githubusercontent.com/47832594/88004513-7f2e9880-cb4a-11ea-867f-38ddd5d1575b.gif)

**Comment by danimateo** [comment]
Another option would be autocomplete scripts for zsh/fish/bash. That would be nice and better in my opinion.

**Comment by RyanGreenup** [comment]
Yeah I agree, I haven't done that before that's why I played with this approach. 

Did you want to have a go at that?

**Comment by danimateo** [comment]
I'll try and make it for zsh and bash at least, haven't used fish but I'll have a look.

**Comment by RyanGreenup** [comment]
If you can do bash and zsh I'll figure out fish, just gotta read up on it.
